src/main/java/com/codejoust/main/service/ProblemService.java

```diff
@@ -76,6 +83,9 @@ public ProblemDto createProblem(CreateProblemRequest request) {
             }
         }
 
+        // Set the owner of this problem
+        problem.setOwner(accountRepository.findAccountByUid(uid));
```

Comment:
If the token is invalid, I think the call to `service.verifyToken` will throw an exception and return an auth error to the user, so this wouldn't be reached - the `verifyToken` method also makes a call to `createAccountIfNoneExists` which does the `accountRepository.findAccountByUid(uid)` null check and creates the account if it doesn't exist

I can add a comment explaining this invariant though since maybe it could be overlooked/confusing if forgotten about in the future